components/stores/moneyStore.js

```diff
@@ -0,0 +1,17 @@
+import { create } from "zustand";
+import { persist, createJSONStorage } from "zustand/middleware";
+
+export const useMoneyStore = create(
+  persist(
+    (set) => ({
+      money: 1000,
+      addMoney: (amount) => set((state) => ({ money: state.money + amount })),
+      subtractMoney: (amount) =>
+        set((state) => ({ money: state.money - amount })),
+    }),
+    {
+      name: "money",
+      storage: createJSONStorage(() => localStorage),
```

Comment:
```suggestion
```

This key is used to switch to sessionStorage, by default it is set to localStorage, so you can just remove it.